docs/content/en/docs/install/reqs.md

```diff
@@ -24,8 +24,21 @@ Kustomize Version: v4.5.4
 Server Version: v1.24.0
 ```
 
-Keptn is not currently compatible with
-[vcluster](<https://github.com/loft-sh/vcluster>).
+Keptn uses a different scheduling method
+when using Kubernetes versions 1.26 and older
+or if the `schedulingGatesEnabled` Helm chart value is set to `false`
+for Kubernetes versions greater than 1.26.
+ See
+[Keptn integration with Scheduling](../architecture/components/scheduler/)
+for details.
+
+Note that you must modify your vCluster configuration
```

Comment:
This paragraph almost seems like a duplicate of the previous one 🤔 
Maybe they can be combined?

Reply:
They are two different statements, actually, but they do look similar.  It's because of the long string about the Helm chart value that implements the "old" scheduler...  And I don't know how to say that more succinctly.  So the logical flow for the section is:

* Keptn works on all k8s 0.24 and later
* The scheduler implementation is different on 0.26 and later, and this links to the Architecture section about scheduling
* vCluster must be modified for k8s before 0.26, and links to instructions for doing this.

Is there a succinct name for the two schedulers?

Reply:
Okay, I tried something different.  I may have used the wrong terminology but see if this works better, and if you can correct my terminology ;-)